packages/amplify_auth_cognito/example/ios/Runner.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,6 +2,6 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:Runner.xcodeproj">
```

Comment:
What is the purpose of this change?

Reply:
I don't know.  It keeps on happening whenever I run on Xcode.  I can remove for now.